### DISCUSSION TITLE
rate limiting

### DISCUSSION THREAD

**Discussion by MovingToContainers** [Netflix/dgs-framework#1515] [Q&A]
HI Team,

Recently from our trainer we got to know that DGS has limitations on the below, can some one help on this.

1) Does DGS has limitation of 10 TPS for instance
2) Does DGS has the limitation  on request payload size up to 8kb.

**Comment by srinivasankavitha** [comment]
The framework does not impose any extra limits on the request payload size. We also don't do any rate limiting. Any behavior wrt payload sizes are enforced by spring boot by default.
